### PR TITLE
updated documentation of cisco gateway

### PR DESCRIPTION
**Pull request by MathieuMonneret**

### CODE REVIEW

File: _content/gateways/cisco/setup.md [hunk 1]
```diff
@@ -153,6 +153,8 @@ gps ubx enable # Enable GPS
 exit # Exit global configuration mode
 ```
 
+> 📜 This command may return the message `packet-forwarder firmware is not installed` this is totally normal and should be ignored.
```
Comment (KrishnaIyer) [inline _content/gateways/cisco/setup.md]: I wouldn't consider this totally normal. Please reword to `This message can be ignored`.

**Review by KrishnaIyer — CHANGES_REQUESTED** [review]
Can you also add the steps to fix the DNS lookup issue?

File: _content/gateways/cisco/setup.md (hunk 1 above)
Comment (MathieuMonneret) [reply _content/gateways/cisco/setup.md]: as the problem is only for private networks i didn't add it...

**Comment by johanstokking** [comment]
@MathieuMonneret  please re-request review when ready

**Review by KrishnaIyer — APPROVED** [review]
Ok changes look good to me. Nice job.

Please fixup your commits before merging.

**Pull request merged**